ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java

```diff
@@ -7940,6 +7970,46 @@ protected Operator genFileSinkPlan(String dest, QB qb, Operator input)
     return output;
   }
 
+  private Path getCTASDestinationTableLocation(CreateTableDesc tblDesc, boolean enableSuffixing) throws SemanticException {
```

Comment:
could we rename it to `getCtasLocation`, to follow same naming pattern `getDefaultCtasLocation`

Reply:
Updated.